q2_moshpit/plugin_setup.py

```diff
@@ -531,6 +533,24 @@
                 "eggnog_proteins.dmnd.gz"
 )
 
+plugin.methods.register_function(
+    function=q2_moshpit.eggnog.fetch_eggnog_proteins,
+    inputs={},
+    parameters={},
+    outputs=[("eggnog_proteins", ReferenceDB[EggnogSequenceTaxa])],
```

Comment:
This will need an updated after you change the corresponding type in https://github.com/bokulich-lab/q2-types-genomics/pull/69